src/routes/(inner)/api/use-click/data.ts

```diff
@@ -0,0 +1,38 @@
+import type { TableData } from '$docs/types.js';
+
+// Options
+export const tableOptions: TableData[] = [
+	{
+		property: 'enabled',
+		description: 'Whether the Hook is enabled, including all internal Effects and event handlers.',
+		type: 'boolean',
+		default: 'true',
+	},
+	{
+		property: 'event',
+		description:
+			'The type of event to use to determine a "click" with mouse input. Keyboard clicks work as normal.',
+		type: '"click" | "mousedown"',
+		default: "'click'",
+	},
+	{
+		property: 'toggle',
+		description: 'Whether to toggle the open state with repeated clicks.',
+		type: 'boolean',
+		default: 'true',
+	},
+	{
+		property: 'ignoreMouse',
+		description:
+			'Whether to ignore the logic for mouse input (for example, if `useHover()` is also being used). When `useHover()` and `useClick()` are used together, clicking the reference element after hovering it will keep the floating element open even once the cursor leaves. This may not be desirable in some cases.',
```

Comment:
This is not something we have to handle in *this* PR, but we should review options for how we handle rich formatting in these description blocks. Those markdown backticks feel very out of place.

Note that Svelte 5 is a lot more strict when it comes to the use of `@html` (as it should be), so we likely can't rely on that as a drop in solution.

Reply:
I'll leave it for now since I'm not entirely sure what you mean. Just FYI I copied these descriptiosn from React's port.